app/serializers/articles_index_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class ArticlesIndexSerializer < ActiveModel::Serializer
+  attributes :id, :title, :lead, :content
+end
```

Comment:
In your client PR, which has been merged, you are not making use of the content data and you are not expecting that in the fixture response. So please remove it from here.

Reply:
We discussed about that. We did not have it and in the group talk it was decided to add it

Reply:
It was during the review with the frontend team  that we mentioned that they should add it to the API since we realised we forgot to add it during the planning. But it is true, we should've added it in the client as well.